Makefile

```diff
@@ -789,6 +795,18 @@ keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_de
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: cinder_kuttl_run
+cinder_kuttl_run: ## runs kuttl tests for the cinder operator, assumes that everything needed for running the test was deployed beforehand.
+	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${CINDER_KUTTL_CONF} ${CINDER_KUTTL_DIR}
+
+.PHONY: cinder_kuttl
+cinder_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy rabbitmq rabbitmq_deploykeystone keystone_deploy keystone_deploy_validate cinder_deploy_prep cinder mariadb_deploy_validate ## runs kuttl tests for the cinder operator. Installs openstack crds and cinder operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
There is a space missing between `rabbitmq_deploy` and `keystone`

Reply:
Also there is no point in having `keystone_deploy_validate` as a dependency since currently it does not work because kuttl does not support scripts in the `assert` command [(they should soon)](https://github.com/kudobuilder/kuttl/issues/221)

Reply:
KUTTL does actually already support scripts in `TestAssert`.  See this as an example: https://github.com/openstack-k8s-operators/osp-director-operator/blob/1dfc187eed383ffc7425ecc3cf065d79964638e0/tests/kuttl/tests/openstackbaremetalset_scale/09-assert.yaml#L36-L52

Reply:
@abays but does this assert work if passed to `kubectl-kuttl assert` command? What I meant is that scripts in TestAssert if fine when running the tests, but not with the assert command.

Reply:
Ah, I have tried that myself.  Fair point.